storybook/pages/CurrencyAmountInputPage.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Storybook 1.0
+import utils 1.0
+
+import shared.controls 1.0
+
+SplitView {
+    id: root
+
+    orientation: Qt.Horizontal
+
+    Item {
+        SplitView.fillWidth: true
+        SplitView.fillHeight: true
+
+        CurrencyAmountInput {
+            id: input
+            anchors.centerIn: parent
+            currencySymbol: ctrlCurrencySymbol.text
+            decimals: ctrlDecimals.value
+            readOnly: ctrlReadOnly.checked
+            enabled: ctrlEnabled.checked
+            value: ctrlAmount.text
+        }
+    }
+
+    LogsAndControlsPanel {
+        SplitView.minimumWidth: 300
+        SplitView.preferredWidth: 400
+
+        SplitView.fillHeight: true
+
+        ColumnLayout {
+            Layout.fillWidth: true
+            RowLayout {
+                Label {
+                    text: "Value:\t"
+                }
+                TextField {
```

Comment:
Using `CurrencyAmountInput` component causes breaking binding with this text field and changing content here does nothing.

Reply:
It works here both ways, if you press ENTER or TAB away

Reply:
This happens when the internal validator gets into this broken state; no idea what's causing this or how to prevent it yet